exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/types/UnsignedColumnTypeTests.kt

```diff
@@ -297,7 +297,7 @@ class UnsignedColumnTypeTests : DatabaseTestsBase() {
 
     @Test
     fun testMaxUnsignedTypesInMySql() {
-        withDb(listOf(TestDB.MYSQL, TestDB.MARIADB)) {
+        withDb(listOf(TestDB.MYSQL_V5, TestDB.MARIADB)) {
```

Comment:
This shouldn't be failing with MYSQL_V8.

Reply:
Actually, it works for all DBs except Postgres >_<